src/Microsoft.DotNet.Darc/Darc/Operations/GetDefaultChannelsOperation.cs

```diff
@@ -77,4 +78,17 @@ public override async Task<int> ExecuteAsync()
             return Constants.ErrorCode;
         }
     }
+
+    private async Task<IEnumerable<DefaultChannel>> FilterDefaultChannels() => (await _barClient.GetDefaultChannelsAsync())
+                    .Where(defaultChannel =>
+                    {
+                        return  (_options.Ids == null || !_options.Ids.Any() || _options.Ids.Any(id => id.Equals(defaultChannel.Id.ToString()))) &&
```

Comment:
Should we compare the guids case insensitive too?

Reply:
I wonder why GUIDs were chosen for subscriptions, but not for anything else.

Reply:
Maybe something to do with the actor state keys